module/Company/view/company/admin/index.phtml

```diff
@@ -24,110 +24,141 @@
         </div>
         <hr>
         <div class="row">
-            <div class="col-md-2">
-                <a class="btn btn-success" href='<?= $this->url('admin_company/default', ['action' => 'addCategory'])?>'> <i class="fas fa-plus"></i><?=$this->translate('Add Category');?>
- </a>
+            <div class="col-sm-6">
+                <a class="btn btn-success" href='<?= $this->url('admin_company/default', ['action' => 'addCategory'])?>'> <span class="glyphicon glyphicon-plus"></span><?=$this->translate('Add Category');?>
```

Comment:
We switched to FontAwesome right? Shoudl probably adjust this

Reply:
Yes, they are already the FA equivalent on master, so they have to be changed "back" (to FA).